Demo/Demo.xcodeproj/project.pbxproj

```diff
@@ -992,6 +992,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
+					"$(FRAMEWORK_SEARCH_PATHS)",
```

Comment:
I found a few threads in the GitHub actions issues that indicated that at times the CI runner cannot find the frameworks, so adding this to the demo app should increase the search path coverage in addition to the above paths.